### ISSUE TITLE
[BUG] Not working with Thunderbird on Manjaro

### ISSUE THREAD

**Issue by mike-lloyd03** [darkreader/darkreader#5541]
# Bug Report

## Describe the bug
With a fresh install of Manjaro KDE and Thunderbird, Dark Reader has no effect on the message preview page or in a message tab.

## Steps to reproduce
I had Dark Reader working properly with Thunderbird. I reinstalled the OS (importing no settings from the previous install) but installing Dark Reader seems to be doing nothing. Thunderbird was installed using pacman and Dark Reader was installed via the Thunderbird extensions tab.

I have tried:
- Toggling the extension on and off
- Adjusting all of the settings (brightness, contrast, sepia...)
- Changing Thunderbird's color settings (Preferences > General > Language & Appearance > Colors)
- Changing the filter mode.

The only setting that has any effect whatsoever is changing the browser theme to custom but that only changes everything outside of the preview pane. The message itself stays light.

Also, I noticed that switching the filter mode away from dynamic and back to dynamic shows a small red circle with an exclamation point in it on the "Dev tools" button. But clicking it doesn't show any kind of warnings, just the CSS editor.

## Expected behavior

Email messages should be dark.

## Actual behavior

Email messages are not dark.

## Screenshots

![image](https://user-images.githubusercontent.com/49411532/114344411-a1bb7400-9b14-11eb-9aee-6834dcce89d5.png)
![image](https://user-images.githubusercontent.com/49411532/114344719-2efec880-9b15-11eb-8333-5f7edee274dc.png)

## System Information

- OS: Manjaro KDE, Linux Kernel 5.11.10
- Browser: Thunderbird 78.9.1 (64-bit)
- Dark Reader Version: 4.9.28

**Comment by wormsky** [comment]
I have the same issue

- OS: Debian (sid)
- Thunderbird: 78.10.0
- Dark Reader: 5.10.0-6-amd64

I found [this reddit topic](https://www.reddit.com/r/Thunderbird/comments/n4k64i/the_dark_reader_add_on_is_not_working_today_do/), linking to [this issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1707573), which might point toward the problem.

**Comment by Gusted** [comment]
That bug report seems to fit the description of the other Thunderbird related issues and is very similar experience I had with DR + TB.

**Comment by karolyi** [comment]
it seems that it's not the responsibility of DR to fix this: https://bugzilla.mozilla.org/show_bug.cgi?id=1707573

**Comment by Gusted** [comment]
> it seems that it's not the responsibility of DR to fix this: https://bugzilla.mozilla.org/show_bug.cgi?id=1707573

Correct, it was pointed in https://github.com/darkreader/darkreader/issues/5541#issuecomment-836956935

**Comment by reisig7** [comment]
I was affected by this bug in Thunderbird 78.14.0 (EOS!) + Dark Reader 4.9.39. (Ubuntu 20.04)
I don't know why, but with Thunderbird 91.4.0 + Dark Reader 4.9.39 it works normally (on the previously affected profile). The referenced TB bug seems to have no activity.
